workspaces/agent-forge/plugins/agent-forge/src/extensions.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { AppRootElementBlueprint } from '@backstage/frontend-plugin-api';
+import React from 'react';
+import ChatAssistantApp from './components/ChatAssistantApp';
+
+export const chatbotDisplayAppRootElement = AppRootElementBlueprint.make({
```

Comment:
This one has me a bit confused, this would be for the New Frontend System which is currently in alpha. I might leave this out for now.